lib/index.ts

```diff
@@ -79,7 +90,11 @@ async function buildDepTreeFromFiles(
   if (_.endsWith(lockFilePath, 'package-lock.json')) {
     lockFileType = LockfileType.npm;
   } else if (_.endsWith(lockFilePath, 'yarn.lock')) {
-    lockFileType = LockfileType.yarn;
+    if (fs.existsSync(path.resolve(root, lockFilePath.replace('yarn.lock', '.yarnrc.yml')))) {
```

Comment:
is there anywhere else these can live? they are always in the same directory?

Reply:
No, they are in the same directory